core_backend/app/routers/question_answer.py

```diff
@@ -57,10 +58,11 @@ async def llm_response(
     return response
 
 
-@classify_safety
-@translate_question
+@check_align_score
```

Comment:
I had the ordering of these wrong. So `main` is actually broken rn. This fixes it.